src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -48,8 +48,9 @@
               :nudgeTop="85"
               :minDate="startDateMinStr"
               :maxDate="startDateMaxStr"
-              :inputRules="getShowErrors ? startDateRules : []"
+              :inputRules="startDateRules || []"
```

Comment:
You don't need the fallback value since startDateRules is a getter and always has a value.